packages/flex-plugin-scripts/src/scripts/__tests__/build.test.ts

```diff
@@ -0,0 +1,119 @@
+import { Environment } from 'flex-dev-utils/dist/env';
+import * as env from 'flex-dev-utils/dist/env';
+import { Bundle } from '../build';
+
+import * as buildScript from '../build';
+import * as prints from '../../prints';
+
+// jest.mock('webpack');
```

Comment:
remove

Reply:
Okay will merge and push to `webpack` branch directly